lib/setup/setupBackup.js

```diff
@@ -649,7 +649,7 @@ class BackupRestore {
     validateBackup(name) {
         return new Promise(resolve => {
             let backups;
-            if (!name && name !== 0) {
+            if (name === undefined || (!name && name !== 0)) {
```

Comment:
I don't understand what this change does.
`!name` is already true for `null`, `undefined`, `""`.

Reply:
@Apollon77 added this line while merging and removed it in the next commit :man_shrugging:

Reply:
My fault ... I also changed in one of my last commits that stupid ... and fixed it after I saw the real problem in @foxriver76 commit, so I fixed it on both places again